browser/app/profile/firefox.js

```diff
@@ -1399,6 +1399,9 @@ pref("browser.uiCustomization.state", "");
 // A restart is mandatory after flipping that preference.
 pref("identity.fxaccounts.enabled", true);
 
+// Use of the Rust backend vs JS backend
+pref("identity.fxaccounts.useRustBackend", false);
```

Comment:
this was the name we came up with during the initial review, suggestions still welcome